### PR TITLE
Easify use CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by YaroslavPodorvanov**
PMM-0

Build: SUBMODULES-0

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -28,6 +28,15 @@ release:                        ## Build static pmm-agent release binary (Linux
...
 	-ldd $(PMM_RELEASE_PATH)/pmm-agent
 
+download-exporters:
+	cd .. && git clone https://github.com/percona/node_exporter || cd ./node_exporter && git pull
+	cd .. && git clone https://github.com/percona/mysqld_exporter || cd ./mysqld_exporter && git pull
+	cd .. && git clone https://github.com/percona/rds_exporter || cd ./rds_exporter && git pull
+	cd .. && git clone https://github.com/percona/postgres_exporter || cd ./postgres_exporter && git pull
+	cd .. && git clone https://github.com/percona/mongodb_exporter || cd ./mongodb_exporter && git pull
+	cd .. && git clone https://github.com/percona/proxysql_exporter || cd ./proxysql_exporter && git pull
+	ls -la .. | grep _exporter
```
Comment (BupycHuk) [inline Makefile:38]: probably it can be replaced by downloading latest version of exporter binaries instead of cloning repositories

Comment (YaroslavPodorvanov) [reply Makefile:38]: adding this part for faster development for beginner in PMM project like me

Comment (ritbl) [reply Makefile:38]: @YaroslavPodorvanov  I agree with @BupycHuk, it is up to a developer what way they want to pull exporters, via `source code` or binaries. If you clone code you might need extra configuration to build/install it.

Comment (ritbl) [reply Makefile:38]: Additionally, you need to install it, just clone it.
So that it is available in `$PATH` (via `$GOPATH/bin`)

**Review by ritbl — CHANGES_REQUESTED** [review]
please see my comments

**Comment by YaroslavPodorvanov** [comment]
Will re-open after full understand project need

**Pull request closed without merging**